### ISSUE TITLE
macOS keep screen awake seems backwards

### ISSUE THREAD

**Issue by mikeckennedy** [np-8/wakepy#13]
It looks like the code for keeping the screen awake is backwards. See the details for caffinate:

**-d      Create an assertion to prevent the display from sleeping**

And the code (ARGS is `-d -u`):

```python
    if keep_screen_awake:
        global _process
        _process = Popen([COMMAND], stdin=PIPE, stdout=PIPE)
    else:
        _process = Popen([COMMAND] + ARGS, stdin=PIPE, stdout=PIPE)
```

It's sending -d -u if we do NOT want to keep the screen awake. I'll send a PR.

**Comment by np-8** [comment]
Yeah I think it looks like it is backwards as you said. One thing I also consider from

```
   -u      Create an assertion to declare that user is active.
           If the display is off, this option turns the display on and prevents the display from going
           into idle sleep. If a timeout is not specified with '-t' option, then this assertion is
           taken with a default of 5 second timeout.
```

Is says that the `-u` flag has by default timeout of 5 seconds. Perhaps there should be something like `-t 999999` added? (~11.5days)

**Comment by np-8** [comment]
Released as part of wakepy 0.4.4 on PyPI. Thanks for reporting and providing PR!

**Comment by mikeckennedy** [comment]
very nice, thanks!

Best,
Michael

Sent via Superhuman ***@***.***>
